simtools/db_handler.py

```diff
@@ -219,33 +216,94 @@ def getModelParameters(
         dict containing the parameters
         '''
 
-        _modelVersion = self._convertVersionToTagged(
-            modelVersion,
-            DatabaseHandler.DB_CTA_SIMULATION_MODEL
-        )
-
         _siteValidated = names.validateSiteName(site)
-        _versionValidated = names.validateModelVersionName(_modelVersion)
         _telModelNameValidated = names.validateTelescopeModelName(telescopeModelName)
 
         if cfg.get('useMongoDB'):
+
+            # Only MongoDB suppports tagged version
+            _modelVersion = self._convertVersionToTagged(
+                modelVersion,
+                DatabaseHandler.DB_CTA_SIMULATION_MODEL
+            )
+            _versionValidated = names.validateModelVersionName(_modelVersion)
+
             _pars = self._getModelParametersMongoDB(
                 DatabaseHandler.DB_CTA_SIMULATION_MODEL,
                 _siteValidated,
                 _telModelNameValidated,
                 _versionValidated,
-                runLocation,
                 onlyApplicable
             )
             return _pars
         else:
+            _versionValidated = names.validateModelVersionName(modelVersion)
+
             return self._getModelParametersYaml(
                 _siteValidated,
                 _telModelNameValidated,
                 _versionValidated,
                 onlyApplicable
             )
 
+    def exportModelFiles(self, parameters, dest):
+
+        def isfloat(value):
+            try:
+                float(value)
+                return True
+            except ValueError:
+                return False
+
+        if cfg.get('useMongoDB'):
+            self._logger.debug('Exporting model files from MongoDB')
+            for par, info in parameters.items():
+                if not info['File']:
+                    continue
+                file = self._getFileMongoDB(
+                    DatabaseHandler.DB_CTA_SIMULATION_MODEL,
+                    info['Value']
+                )
+                self._writeFileFromMongoToDisk(DatabaseHandler.DB_CTA_SIMULATION_MODEL, dest, file)
+        else:
+            self._logger.debug('Exporting model files from local model file directories')
+            for par, value in parameters.items():
+
+                shallNotSearch = (
```

Comment:
In order to check if a parameter is a file, I check either

`if any(ext in str(value) for ext in ['.dat', '.txt', '.lis']):`

or 

`if '.dat' in str(value) or '.txt' in str(value):`

I think we should make it a function in db_handler and use the same one everywhere. 
Either way, "shallNotSearch" is not the best name, I would call it "isFile".

Reply:
Where is this function that you use? I agree we should use the same here

Reply:
I created the function _isFile and replaced the code.

Reply:
I use it twice, once in 
https://github.com/gammasim/gammasim-tools/blob/9f6ab59429fbbeab8b2265660f3d1ac1ac35ebc7/simtools/db_handler.py#L931

and another time in "updateParameter", but that code is not pushed yet since I didn't finish working on it. Now that you made these changes, I will have to rewrite it a bit, but that's OK. If you change the line above, we can close the conversation and I will make sure to use the function in my next update.

Reply:
What line above?
I created the _isFile function as you suggested.

Reply:
Line 931 showed above. That one should be changed to use the new `_isFile` method.